tests/test_project_access.py

```diff
@@ -14,11 +14,43 @@
 # proj_data = {"pi": "piName", "title": "Test proj", "description": "A longer project description"}
 proj_query = {"project": "public_project_id"}
 # proj_query_restricted = {"project": "restricted_project_id"}
-
+first_new_email = {"email": "first_test_email@mailtrap.io"}
+first_new_user = {**first_new_email, "role": "Researcher"}
 
 # UTILITY FUNCTIONS ############################################################ UTILITY FUNCTIONS #
 
 
+def get_existing_projects():
+    """Return existing projects for the tests"""
+    existing_project_1 = models.Project.query.filter_by(public_id="public_project_id").one_or_none()
+    existing_project_2 = models.Project.query.filter_by(
+        public_id="second_public_project_id"
+    ).one_or_none()
+
+    return [existing_project_1, existing_project_2]
```

Comment:
Could technically just do this --> tuple instead of list. But doesn't matter
```suggestion
    return existing_project_1, existing_project_2
```